README.md

```diff
@@ -507,6 +508,8 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
       - mini.5p--3p.bam: subsample of hifi reads from the public pacbio dataset [pbmc singlecell mini](https://downloads.pacbcloud.com/public/dataset/IsoSeq_sandbox/2022_pbmc_singlecell_mini/ccs.bam) where the primers where removed with LIMA.
       - test.sorted.bam: 35 reads from a [public PacBio Revio dataset of HG002](https://downloads.pacbcloud.com/public/revio/2022Q4/HG002-rep1/analysis/HG002.m84011_220902_175841_s1.GRCh38.bam) aligned to GRCh38.
       - test.sorted.bam.bai: BAM index for 'test.sorted.bam'
+      - NA03697B2_downsampled.pbmm2.repeats.bam: subsample of puretarget pacbio reads from the public pacbio dataset aligned to genome3.fasta
```

Comment:
Can you give a link to the public pacbio dataset website/download link and you're good to go :)

Reply:
done! :) thanks!